string ip
int32 port
int32 slave_id
int32[] is_rtu

---
int32 res
int32 index